lib/ansible/modules/windows/win_scheduled_task.py

```diff
@@ -99,4 +119,46 @@
     state: present
     enabled: yes
     user: SYSTEM
+
+# Create a scheduled task to run a PowerShell script
```

Comment:
Make this the `name:` field of the task, we like to give every example a proper name as we consider this good practice we should do it ourselves too ;-)